### PR TITLE
GA MOBILE_USER_DEMO_MODE, merge into PRACTICE_MOBILE_WORKERS priv

### PR DESCRIPTION
**Pull request by sravfeyn**
## Product Description
https://dimagi-dev.atlassian.net/browse/SC-2475

All that this toggle `MOBILE_USER_DEMO_MODE` did was allow toggling the demo mode on/off, rest of the feature was under a paid privilege called `PRACTICE_MOBILE_WORKERS`, so this PR merges the toggle into privilege

## Feature Flag

Removes MOBILE_USER_DEMO_MODE

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by Charl1996 — APPROVED** [review]

**Comment by Charl1996** [comment]
@sravfeyn I presume there are no domains on Community Plan that will now not have this privilege?

**Comment by sravfeyn** [comment]
Good question, Chris. I have verified that none of the community domains that had this flag enabled have a demo user turned on, so it should be safe to remove access to them..

```
In [16]: for d in domains:
    ...:     any([d for u in Domain.get_by_name(d).all_users() if getattr(u, 'is_demo_user', False)
    ...: ])
```

**Comment by Charl1996** [comment]
Cool, then I suppose this can be merged providing the failed test passes.

File: corehq/toggles/__init__.py [hunk 1]
```diff
@@ -2472,3 +2465,12 @@ def domain_has_privilege_from_toggle(privilege_slug, domain):
     TAG_SOLUTIONS_CONDITIONAL,
     [NAMESPACE_DOMAIN]
 )
+
+MOBILE_USER_DEMO_MODE = FrozenPrivilegeToggle(
```
Comment (Charl1996) [inline corehq/toggles/__init__.py:2469]: How is this actually used now, since the toggle is moved under the PRACTICE_MOBILE_WORKERS privilege?

Comment (Charl1996) [reply corehq/toggles/__init__.py:2469]: Oh, wait, nevermind, I click that you're doing this to associate the privilege with the toggle.

**Pull request merged**